qa/os/src/test/java/org/elasticsearch/packaging/util/Docker.java

```diff
@@ -487,7 +487,7 @@ private static void verifyOssInstallation(Installation es) {
             .forEach(
                 cliBinary -> assertTrue(
                     cliBinary + " ought to be available.",
-                    dockerShell.runIgnoreExitCode("hash " + cliBinary).isSuccess()
+                    dockerShell.runIgnoreExitCode("bash -c  'hash " + cliBinary + "'").isSuccess()
```

Comment:
To be honest, I'm not sure why this change is necessary. Something about the default shell might be different? Other command work just fine but for some reason if I just do `hash foo` via `docker exec` it complains that it can't find a builtin by that name. Why it's look for a builtin command I have no clue. In the end this change doesn't affect the purpose of the assertion, which is to validate these tools are available in the container.

Reply:
Wow, that's really peculiar...but ultimately it doesn't matter, as you say.